docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -176,33 +113,156 @@ This example configuration file accepts input from an OpenTelemetry Agent, proce
-see the {ot-collector}[OpenTelemetry collector] docs.
+By defining the following environment variables, you can customize the OTLP endpoint the agent will use to communicate with
+APM Server.
+
+[source,bash]
+----
+export OTEL_RESOURCE_ATTRIBUTES=service.name=frontend,service.version=1.0-SNAPSHOT,deployment.environment=staging
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
+java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
+     -jar target/frontend-1.1.jar
+----
+
+|=== 
+
+| `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
+
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
+
+|===
+
+You are now ready to collect <<open-telemetry-elastic-traces-metrics,traces and metrics>>, <<open-telemetry-elastic-verify,verifying metrics>>,
+and <<open-telemetry-elastic-kibana,visualizing metrics>> in {kib}.
+
+[[open-telemetry-elastic-protocol]]
+==== APM Server native support of OpenTelemetry protocol
+
+experimental::[This feature is experimental and may be changed in a future release. It is only available in a self-managed environment.]
+
+The APM Server native support of OpenTelemetry protocol integration allows you to send collected telemetry data directly from your applications to APM Server.
+Trace data collected from your services and the metrics data collected from your applications and infrastructure are sent using the
+OpenTelemetry protocol.
+
+image::images/open-telemetry-protocol-arch.png[OpenTelemetry Elastic protocol architecture diagram]
+
+[float]
+[[instrument-apps-apm-server]]
+===== Instrument applications
+
+To export traces and metrics to APM Server, ensure that you have instrumented your services and applications
+with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to execute your Java app using the
+https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry agent for Java]. 
+
+By defining the following environment variables, you can customize the OTLP endpoint so that the OpenTelemetry agent communicates with
+APM Server.
+
+[source,bash]
+----
+export OTEL_RESOURCE_ATTRIBUTES=service.name=checkoutService,service.version=1.1,deployment.environment=production
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer apm_secret_token"
+java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
+     -classpath lib/*:classes/ \
+     com.mycompany.checkout.CheckoutServiceServer
+----
+
+|=== 
+
+| `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
+
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
+
+| `OTEL_EXPORTER_OTLP_HEADERS` | Authorization header that includes the Elastic APM Secret token or API key. Please note the required space between `Bearer` and `apm-secret-token`.
```

Comment:
Perhaps provide an example of how to format an API Key too? And/or link to https://www.elastic.co/guide/en/apm/server/current/api-key.html

Reply:
Very good point. I thought about it and dropped the ball. 

Would it be like 
```
export OTEL_EXPORTER_OTLP_HEADERS="Authorization=ApiKey R0gzRWIzUUI3eVpiU054S3pYSy06bXQyQWl4TlZUeEcyUjd4cUZDS0NlUQ=="
```

Reply:
@cyrille-leclerc yes that looks correct.

Re additional headers, I'm onboard with having method-specific headers. I'd still like to wait for feedback like we discussed recently.

Reply:
@EamonnTP can you please document the usage of API akei based authentication with:
``` 
export OTEL_EXPORTER_OTLP_HEADERS="Authorization=ApiKey R0gzRWIzUUI3eVpiU054S3pYSy06bXQyQWl4TlZUeEcyUjd4cUZDS0NlUQ=="
```

@axw are we sure that we can't simplify and also handle the api key authentication with the `OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer xxx` format?

Reply:
@cyrille-leclerc are you asking if we can support an additional auth header as well as Authorization? Yes we can. Maybe you could open an issue describing how you would like it to look, and we can discuss further there?

Reply:
I was not clear, my question is: can we use the same `Authorization=Bearer xxx` format for both Secret Token authentication and API Key authentication.
We would support `Authorization=Bearer xxx` with `xxx` being the "secret token" or the "api key"

Reply:
Oh I see. `Authorization: ApiKey <api_key>` was chosen to be consistent with Elasticsearch. We could change that, but that's an even bigger discussion.

Reply:
If it's a bigger discussion, let's park it for the moment :-) We will solve the challenge of clarity primarily with documentation (and hopefully with error messages)

Reply:
I updated this section. Hopefully it provides more clarity.